frontend/src/components/Home.tsx

```diff
@@ -0,0 +1,4 @@
+const Home: React.FC = () => {
+  return <h1>Home</h1>;
+};
```

Comment:
i don't think if it's necessary. @nirojneupane10 @ashimvaricon @pradip-varicon What do you guys think?

Reply:
@adars-kafle i have used home.tsx for testing only to test the user and admin role auth.

Reply:
alrighty!